examples/README.md

```diff
@@ -0,0 +1,36 @@
+# Examples
+
+The directory contains simple examples for OTA updates
+
+## beta
+
+This is the example app that will replace the running OTA app. Typically, any new image should contain OTA update logic.
+This app should be built before any of the ota apps can use it. 
+It is better to build in release mode to minify the application. 
+
+Build: 
+```
+RUSTFLAGS="-C strip=symbols -C debuginfo=0" cargo build --example beta --release
```

Comment:
Does the stripping etc really need to be done here? It significantly complicates the example instructions. Is it required to fit on some specific type of chip? Otherwise I think simple instructions is more valuable.

Reply:
Doesn't need to happen, but it is an important aspect of OTA updates in my experience. We should mention it in the docs if not as part of the example, but it does greatly speed up the flashing.